src/Elasticsearch.Net/Api/RequestParameters/RequestParameters.NoNamespace.cs

```diff
@@ -1248,17 +1241,10 @@ public class MultiSearchRequestParameters : RequestParameters<MultiSearchRequest
 		}
 	}
 
-	///<summary>Request options for MultiSearchTemplate <para>http://www.elastic.co/guide/en/elasticsearch/reference/current/search-multi-search.html</para></summary>
+	///<summary>Request options for MultiSearchTemplate <para>https://www.elastic.co/guide/en/elasticsearch/reference/current/search-multi-search.html</para></summary>
 	public class MultiSearchTemplateRequestParameters : RequestParameters<MultiSearchTemplateRequestParameters>
 	{
 		public override HttpMethod DefaultHttpMethod => HttpMethod.POST;
-		///<summary>Indicates whether network round-trips should be minimized as part of cross-cluster search requests execution</summary>

```

Comment:
breaking change?

Reply:
yes, looks to have been removed from the spec:

- 7.3: https://github.com/elastic/elasticsearch/blob/7.3/rest-api-spec/src/main/resources/rest-api-spec/api/msearch_template.json#L44-L48
- 7.4: https://github.com/elastic/elasticsearch/blob/7.4/rest-api-spec/src/main/resources/rest-api-spec/api/msearch_template.json